docs/poc2023q1/edge-scheduler.md

```diff
@@ -0,0 +1,65 @@
+### Edge scheduler
+The edge scheduler monitors the EdgePlacement, Location, and SyncTarget objects and maintains the results of matching.
+
+It starts from the kcp-scheduling-placement controller separated from kcp-dev/kcp, now being refactored towards the goal above.
+
+It works with [this snapshot](https://github.com/kcp-dev/kcp/tree/4506fdc064060b3fe82e1082533f9798b36ba7a5) of kcp.
+
+#### A short demo
+Start a kcp server with [this snapshot](https://github.com/kcp-dev/kcp/tree/4506fdc064060b3fe82e1082533f9798b36ba7a5).
+
+Point `$KUBECONFIG` to the admin kubeconfig for that kcp server.
+
+Workspace `root:edge` is used as the edge service provider workspace.
+```console
+kubectl ws root
+kubectl ws create edge --enter
+```
+
+Create CRDs and expose APIs.
```

Comment:
Perhaps we could say "Install Edge Placement CRDs and API Exports"

Reply:
Fixed in latest push. Since we have two CRDs, I just say "Install CRDs".